lib/rules/web/admin_console/4.3/actions.xyaml

```diff
@@ -1,3 +1,8 @@
+wait_action_button_loaded:
```

Comment:
Maybe just increase the timeout of `click_resource_action_button` ?

Reply:
Updated as suggested, thanks!